src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -217,7 +217,7 @@ private void readHeader() throws IOException {
         PositionalBufferedStream pbs = null;
         try {
             is = ParsingUtils.openInputStream(path);
-            if (isGZIPPath(path)) {
+            if (hasBlockCompressedExtension(path) || hasBlockCompressedExtension(new URI(path))) {
```

Comment:
I don't think so, so I will change it. I was following the same approach as in `TribbleIndexedFeatureReader.readHeader()`. I can change it also there or even better, modify the `AbstractFeatureReader.hasBlockCompressedExtension (final File file)` to be something like:

```java
public static boolean hasBlockCompressedExtension (final File file) {
    return hasBlockCompressedExtension(new URI(file));
}
``